cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/stream/StreamInputFormatProvider.java

```diff
@@ -14,14 +14,16 @@
  * the License.
  */
 
-package co.cask.cdap.data.stream;
+package co.cask.cdap.internal.app.runtime.batch.stream;
```

Comment:
Well I wasn't sure about a good place to put it (since .data package in app-fabric didn't sound right). Open to suggestions.